### PR TITLE
Routing

### PR DESCRIPTION
**Pull request by ivergentz**
Added simple navigation with react router.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -22,7 +23,10 @@
     "build-storybook": "build-storybook -s public"
   },
   "eslintConfig": {
-    "extends": ["react-app", "pliugin:cypress/recommended"]
+    "extends": [
+      "react-app",
+      "pliugin:cypress/recommended"
```
Comment (carolinwagner) [inline package.json]:
```suggestion
      "plugin:cypress/recommended"
```

File: src/App.js [hunk 2]
```diff
@@ -26,4 +48,5 @@ const Pagebody = styled.div`
   background: var(--back-light);
   height: 100vh;
  
-`
\ No newline at end of file
+`
+//grid zum vereinfachen des Layouts
```
Comment (carolinwagner) [inline src/App.js]:
```suggestion

```

File: src/App.js [hunk 3]
```diff
@@ -1,19 +1,41 @@
...
-import DashboardPage from './dashboardPage/DashboardPage'
+import Navbar from './navigation/Navigation'
+import {
+  BrowserRouter as Router,
+Route, 
+Switch,
+} from "react-router-dom"
+import CreatePage from './create/CreatePage'
+import SearchPage from './search/SearchPage'
+import DashboardPage from './dashboardPage/DashboardPage';
```
Comment (carolinwagner) [inline src/App.js]: Inconsistent naming: You could either add "page" to the `create`/`search` folder names, or remove it from the `dashboardPage` folder name

File: src/create/CreatePage.js [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+  <Hello>Hallo</Hello>
+    )}
+
+    const Hello = styled.p`
+    color: white;
+    padding-top: 20px;
+    display: grid;
+    justify-items: center;
+    align-items: center;
+    `
```
Comment (carolinwagner) [inline src/create/CreatePage.js]:
```suggestion
const Hello = styled.p`
  color: white;
  padding-top: 20px;
  display: grid;
  justify-items: center;
  align-items: center;
`
```

File: src/create/CreatePage.js [hunk 5]
```diff
@@ -0,0 +1,16 @@
+import React from 'react'
+import styled from 'styled-components/macro'
+
+export default function Create() {
+
+  return (  
+  <Hello>Hallo</Hello>
+    )}
```
Comment (carolinwagner) [inline src/create/CreatePage.js]: Did you forget to run your formatter? The indentation is a little off...

```suggestion
  return (  
    <Hello>Hallo</Hello>
  )}
```

File: src/random.txt [hunk 6]
```diff
@@ -0,0 +1,29 @@
...
+          <Route path="/users">
+            <Users />
+          </Route>
+          <Route path="/">
+            <Home />
+          </Route>
+        </Switch>
+      </div>
+    </Router>
+  );
```
Comment (carolinwagner) [inline src/random.txt]: Maybe this was just a note for yourself? I guess this should not be checked in, right?

**Review by carolinwagner — CHANGES_REQUESTED** [review]
Generally looks good to me - I added some comments and you should check your formatter (Prettier). Seems like it isn't working properly.

**Review by jamarob — COMMENTED** [review]
Your `.prettierrc` looks okay, you could try the following to make it work again
- check if the extension is enabled
- format on save is true in the editor settings
- command palette -> `format document with` -> `configure default formatter..` -> select prettier

Prettier will not format files that contain syntax error, so check that, too.

**Comment by ivergentz** [comment]
Thanks. Prettier works again and made the suggested changes from Caro.

**Pull request closed without merging**